src/components/Configs/Changes/ConfigChangesGraph.tsx

```
@@ -0,0 +1,129 @@
+  changes: ConfigChange[];
+  onItemClicked?: (change: ConfigChange) => void;
+};
+
+export default function ConfigChangesGraph({
+  changes,
+  onItemClicked = () => {}
+}: ConfigChangesGraphProps) {
+  const data: ChartShallowDataShape[] = useMemo(() => {
+    return changes.map((change) => ({
+      key: dayjs(change.first_observed).toDate(),
+      data: change.config?.name!,
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)